wireapp/wire-android-reloaded
# feat(delete-message): hide edit context menu when delete action triggered [AR-923]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-923" title="AR-923" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-923</a>  Deleting text messages
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues
[AR-923]
### Causes (Optional)
when the user select the delete message item from the long tap edit context menu, the menu was still open!

### Solutions

Hide the menu when the delete message item is triggered.

### Testing

#### Test Coverage (Optional)
We need UI tests, but now it's manual!

#### How to Test

Go to a conversation, long tap on any message, select the delete item, and the menu should disappear before the dialog pops up.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

[AR-923]: https://wearezeta.atlassian.net/browse/AR-923
